### DISCUSSION TITLE
[Next.js] - Using the Sitecore template on Vercel with the JSS front-end developer trial

### DISCUSSION THREAD

**Discussion by cresencio** [Sitecore/jss#1624] [Help] [answered]
### Description

Hello, I just signed up for the front-end trial license provided by Sitecore. Are the api keys compatible with the Vercel template? By that I mean, can I click on Vercel's "deploy" button and use the keys provided by Sitecore. Sorry if that is unclear I am just trying to play around with the demo template and jss.

**Comment by illiakovalenko** [comment, marked as answer]
@cresencio Hey, there are some required environment variables when you deploy _next_.js with _jss_ on Vercel. It includes SITECORE_API_KEY. Sure, you can use keys, and information that are provided by your Sitecore instance.
